manager/plugin/plugin.go

```diff
@@ -173,7 +173,8 @@ func (m *vatzPluginManager) Uninstall(name string) error {
 		}
 		if running {
 			log.Error().Str("module", "plugin").Err(err).Msgf("Plugin %s is currently running, Please, kill plugin first.", name)
-			return nil
+			errorMessage := fmt.Sprintf("Please, kill plugin: %s first.", name)
```

Comment:
How about print out "Please, stop plugin" instead "kill"? because we have stop command in vatz.
It's just an expression. I think anything is okay, it's up to you :)

Reply:
> How about print out "Please, stop plugin" instead "kill"? because we have stop command in vatz. It's just an expression. I think anything is okay, it's up to you :)

Yeah, you are right. stop is right word for the error message.